Source/Controllers/DataExport/SPExportController.m

```diff
@@ -403,6 +403,12 @@ - (IBAction)export:(id)sender
 {
 	SPExportType selectedExportType = SPAnyExportType;
 	SPExportSource selectedExportSource = SPTableExport;
+
+    // if they are exporting and haven't selected a table
+    // loadTableValues will fail, so select the last table
+    if([tablesListInstance selectedTableItems].count == 0){
+        [tablesListInstance selectTableAtIndex:@(tablesListInstance.tables.count-1)];
```

Comment:
Isn't there `[tablesListInstance.tables lastObject]` or something like that?

Reply:
There is, but I need the index, not the object..

Reply:
ohh right